src/templates/powered_by_template.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <head>
+        <title>{{title}}</title>
+        <link rel="stylesheet" type="text/css" href="static/styles.css">
+    </head>
+    <body>
+        <h1>{{title}}</h1>
+        <p>{{num_configs}} collections.</p>
+        <div id="collections">
+            <table>
+                <thead>
+                    <tr>
+                        <th>Identifier</th>
+                        <th>Action</th>
+                        <th>Contact</th>
+                        <th>Description </th>
+                        <th>Enabled</th>
+                        <th>Parameters</th>
+                    </tr>
+                </thead>
+                <tbody class="list">
+                    {% for key, value in data.items() %}
+                        {% if enable_query is none or value.get("enabled") is enable_query %}
+                        <tr>
+                            <td class="identifier">{{key}}</td>
+                            <td class="action">{{value.get("action")}}</td>
```

Comment:
nit: we know that `action` and `enabled` will be there, we can just do `value["action"]` or in Jinja I think you can also do `{{ value.action }}`